lesson3/index_hw2.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="style_hw2.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header class="header">
+			<div class="logo">
+				<img src="logo.png">				
+			</div>
+			<ul class="menu">
+				<li class ="menu__item"><a class="menu__link menu__link--first" href="#">Главная</a></li>
+				<li class ="menu__item"><a class="menu__link" href="#">Товары</a></li>
+				<li class ="menu__item"><a class="menu__link" href="#">О_нас</a></li>					<li class ="menu__item"><a class="menu__link" href="#">Контакты</a></li>
+			</ul>
+		</header>
+		<div class="container">
+			<ul class="aside__menu">
+				<li class="aside__item"><a class="aside__link" href="">Ноутбуки</a></li>
+				<li class="aside__item"><a class="aside__link" href="">Планшеты</a></li>
+				<li class="aside__item"><a class="aside__link" href="">Телефоны</a></li>
+				<li class="aside__item"><a class="aside__link" href="">Телевизоры</a></li>					
+				<li class="aside__item"><a class="aside__link" href="">Бытовая техника</a></li>
+				<li class="aside__item"><a class="aside__link" href="">Автотовары</a></li>					
+			</ul>				
+			<article class="article">
+				<div class="picture">
```

Comment:
I don't like classes picture and photo. Do you really like them? you can have article, article__photo-container, article__photo, article__description. How do you think?